spec/components/collections/version_description_component_spec.rb

```diff
@@ -13,15 +13,15 @@
     let(:collection_version) { build_stubbed(:collection_version) }
 
     it 'does not renders the component' do
```

Comment:
"render" not "renders"

Reply:
Thank you. I've made the fix. Also added the "*" to the expectation.